src/data/directory/lgu/region-i-ilocos-region.json

```
@@ -938,27 +930,27 @@
             "name": "PEDRO A. MERRERA III"
           },
           "vice_mayor": {
-            "name": "SIMPLICIO L. ROSARIO"
+            "name": "EDGAR MAMENTA"
           }
         },
         {
           "municipality": "Bolinao",
           "zip_code": "2406",
           "mayor": {
-            "name": "ALFONSO F. CELESTE"
+            "name": "JESUS 'BOYING' CELESTE"
```

```suggestion
            "name": "JESUS SALVADOR DEL FIERRO CELESTE"
```

ref: https://lgu201.dilg.gov.ph/view.php?r=01&p=055&m=14